automerge/Cargo.toml

```diff
@@ -2,6 +2,7 @@
 name = "automerge"
 version = "0.1.0"
 edition = "2018"
+license = "MIT"
```

Comment:
Should we also add the `LICENSE` file to this repo?

Reply:
Yep, will do